streams/client/client.go

```diff
@@ -50,6 +98,8 @@ var (
 	EstablishStreamTimeout time.Duration = 5 * time.Second
 	SendToPeerTimeout      time.Duration = 10 * time.Second
```

Comment:
Shoud these be exported? Considering we now set these in request, and this is default if not set

Reply:
yes. to alter default